scripts/verify_conda_libmesh.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# * This file is part of the MOOSE framework
+# * https://www.mooseframework.org
+# *
+# * All rights reserved, see COPYRIGHT for full restrictions
+# * https://github.com/idaholab/moose/blob/master/COPYRIGHT
+# *
+# * Licensed under LGPL 2.1, please see LICENSE for details
+# * https://www.gnu.org/licenses/lgpl-2.1.html
+
+import os
+import sys
+try:
+    from conda_build.metadata import MetaData
+except ImportError:
+    print('conda_build module not available.')
```

Comment:
we need to fail gracefully here as not all users will have conda, let's skip the output and return 0, otherwise the whole build will fail